src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,8 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public interface MachineProducer {
-    /**
-     * In your implementations - create 2-3 models, add them to the list and return
-     * @return - the list of models
-     */
-    List<Object> get();
+public interface MachineProducer<T> {
+    List<? extends Machine> get();
```

Comment:
Do not use raw types. The return type of the `get()` method should be parameterized with the same type as the `MachineProducer` interface to ensure type safety.

Reply:
Use T